test/kennel/models/monitor_test.rb

```diff
@@ -155,6 +155,36 @@ def monitor(options = {})
       ).as_json[:options][:no_data_timeframe].must_be_nil
     end
 
+    it "sets renotify_statuses to alert only when renotify_interval is greater than 1, and notify no data and warning are disabled" do
+      monitor(
+        renotify_interval: -> { 10 },
+        notify_no_data: -> { false },
+        ).as_json[:options][:renotify_statuses].must_equal ["alert"]
+    end
+
+    it "sets renotify_statuses to alert and warn when renotify_interval and warning are set" do
+      monitor(
+        renotify_interval: -> { 10 },
+        notify_no_data: -> { false },
+        warning: -> { 10 },
+        ).as_json[:options][:renotify_statuses].must_equal ["alert", "warn"]
```

Comment:
`rubocop -a test/kennel/models/monitor_test.rb`
can also do as part of commit with https://github.com/grosser/dotfiles/blob/master/bin/pre_commit_rubocop.rb